core/src/test/kotlin/com/tschuchort/compiletesting/CompilerPluginsTest.kt

```diff
@@ -66,6 +66,20 @@ class CompilerPluginsTest {
 
         verify(mockPlugin, atLeastOnce()).registerProjectComponents(any(), any())
 
-        Assertions.assertThat(result.exitCode).isEqualTo(KotlinCompilation.ExitCode.OK)
+        Assertions.assertThat(result.exitCode).isEqualTo(ExitCode.OK)
     }
-}
\ No newline at end of file
+
+    @Test
+    fun `when JS compiler plugins are added they get executed`() {
```

Comment:
This comes back to how are we going to verify the correct JS files? I can write some really basic tests, maybe simple text parsing will be sufficient initially.

Reply:
Something basic should be enough for now. Like testing that the expected JS file is created, testing that the text contains a string `"class Foo"` and `"function bar()"`. We don't really have to test that the right code is generated since we're not testing the compiler itself, but a basic test will hopefully catch regressions and edge cases on different platforms or when compiler internals change.